validation/src/test/scala/hmda/validation/engine/lar/quality/LarQualityEngineSpec.scala

```diff
@@ -0,0 +1,26 @@
+package hmda.validation.engine.lar.quality
+
+import java.io.File
+
+import hmda.parser.fi.lar.{ LarCsvParser, LarGenerators }
+import org.scalatest.{ MustMatchers, PropSpec }
+import org.scalatest.prop.PropertyChecks
+
+import scala.io.Source
+
+class LarQualityEngineSpec
+    extends PropSpec
+    with PropertyChecks
+    with MustMatchers
+    with LarGenerators
+    with LarQualityEngine {
+
+  property("A LAR must pass quality checks") {
+    val lines = Source.fromFile(new File("parser/src/test/resources/txt/FirstTestBankData_clean_407_2017.txt")).getLines()
+    val lars = lines.drop(1).map(l => LarCsvParser(l))
+
+    lars.foreach { lar =>
+      checkQuality(lar).isSuccess mustBe true
+    }
```

Comment:
@jmarin is correct. Most filings will have quality and macro quality errors. I cannot separate out quality and macro so these stats have both but here are the numbers of filings with quality/macro edits.

-  2012: 92% of institutions have quality/macro errors, 19% of all LARs
-  2013: 92% of institutions have quality/macro errors, 18% of all LARs
-  2014: 91% of institutions have quality/macro errors, 16% of all LARs
-  2015: 93% of institutions have quality/macro errors, 17% of all LARs